himportscan/src/HImportScan/ImportScanner.hs

```diff
@@ -198,16 +204,25 @@ scanTokenStream fp toks =
     comment :: Parsec [Located Token] () String
     comment =
       satisfyEvenComments "comment" (\case
+#if __GLASGOW_HASKELL__ >= 902
+        ITblockComment c _ -> Just c
+        ITlineComment c _ -> Just c
+#else
         ITblockComment c -> Just c
         ITlineComment c -> Just c
+#endif
```

Comment:
Rather than using CPP, I'd prefer to abstract in `HImportScan.GHC` the differences.